modules/claims_api/app/workers/claims_api/ews_updater.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'bgs'
+require 'claims_api/claim_logger'
+
+module ClaimsApi
+  class EwsUpdater
+    include Sidekiq::Worker
+
+    def perform(bgs_claim, target_veteran, ews)
+      bgs_service = BGS::Services.new(
+        external_uid: target_veteran[:external_uid],
+        external_key: target_veteran[:external_key]
+      )
+      response = bgs_service.benefit_claims.update_bnft_claim(claim: bgs_claim)
+
+      if response[:bnft_claim_dto].nil?
+        ews[:status] = ClaimsApi::EvidenceWaiverSubmission::ERRORED
+        ews.vbms_error_message = "BGS Error: update_birls_record failed with code #{response[:return_code]}"
```

Comment:
Error message seems weird here. assuming  copy / pasta from another? and it's `vbms_error_message` getting `BGS Error:`